app/src/androidTest/java/com/github/houseorganizer/houseorganizer/TaskListOnMainScreenTest.java

```diff
@@ -30,10 +38,15 @@
 
 import java.util.concurrent.ExecutionException;
 
+/*
+* To add tests for: adding/removing tasks, changing task title [for now need bug fixes]
+*/
+
 @RunWith(AndroidJUnit4.class)
 public class TaskListOnMainScreenTest {
 
-    private static FirebaseFirestore db;
+    public static final String NEW_TASK_TITLE = "this is my task now";
+    public static final String NEW_SUBTASK_TITLE = "a new subtask!";
     private static FirebaseAuth auth;
 
     @BeforeClass
```

Comment:
Do you create a new tasklist on firebase in this test ? If so, is it deleted afterwards ?

Reply:
No new task list is created. All actions on the database by this test class are reverted in their respective methods.

Reply:
Oh I see you do it in each test, ok!